crates/cashu/Cargo.toml

```diff
@@ -14,7 +14,7 @@ bench = []
 
 [dependencies]
 uuid = { version = "1", features = ["v4", "serde"], optional = true }
-bitcoin = { version = "0.32.2", features = [
+bitcoin = { version = "0.32.5", features = [
```

Comment:
cargo will automatically pull in the latest patch version so we do not want to raise this as it should be the minimum version we support.  This is true for a few of the ones updated, just going to note it here.